pkg/build/controller/policy/policy_test.go

```diff
@@ -120,3 +120,30 @@ func TestHandleComplete(t *testing.T) {
 		t.Errorf("build-3 should not have Status.StartTimestamp set")
 	}
 }
+
+func TestHandleCompleteParallel(t *testing.T) {
+	builds := []buildapi.Build{
+		addBuild("build-1", "sample-bc", buildapi.BuildPhaseComplete, buildapi.BuildRunPolicyParallel),
```

Comment:
it doesn't really matter, the point is to ensure that when a build completes, we immediately start all queued parallel builds.  the code path is the same regardless of what type of build completed.

but for sanity, i'll change it.